gems/pending/appliance_console/database_configuration.rb

```diff
@@ -257,7 +257,7 @@ def initialize_from_hash(hash)
         else
           raise ArgumentError, "Invalid argument: #{k}"
         end
-      end
+      end unless hash.empty?
```

Comment:
Not after implementing your suggestion: ` Maybe we shouldn't be subclassing DatabaseConfiguration at all.`  :)